pkg/runner/deb/packageconfig.go

```diff
@@ -31,5 +34,48 @@ func (p *PackageConfig) UpdatePackageGroup(g *PackageGroup) error {
 		g.Add(*p.Override...)
 	}
 
+	if p.OverrideDir != nil {
+		overrides := make([]Package, 0, len(g.packages))
+		for _, pkg := range g.packages {
+			if pkg.LocalPath != nil {
+				continue
+			}
+			if localDeb, err := findLocalDeb(pkg, p.OverrideDir); err == nil {
+				overrides = append(overrides, Package{
+					Name:      pkg.Name,
+					LocalPath: &localDeb,
+				})
+			}
+		}
+
+		if len(overrides) > 0 {
```

Comment:
This is minor, but why is `overrides` needed here, if you're just going bulk `Add` them anyway?  You can just `Add` them directly without needing the list and the spread.  The reason the code above creates an overrides list, is because it checks for errors before performing the add, so it needs to collect the package info in a temporary location.